packages/tabs/examples/with-wrappers.example.js

```diff
@@ -0,0 +1,37 @@
+import "@reach/tabs/styles.css";
+
+import React from "react";
+import { Tabs, TabList, Tab, TabPanels, TabPanel } from "@reach/tabs";
+
+export const name = "With Wrappers";
+
+export const Example = () => (
+  <Tabs>
+    <div style={{ display: "flex" }}>
+      <TabList>
+        <Tab>One</Tab>
+        <Tab>Two</Tab>
+        <Tab>Three</Tab>
+      </TabList>
+      <div style={{ marginLeft: 30 }}>
+        Here is content grouped with the tab list
+      </div>
+    </div>
+
+    <div style={{ background: "ghostwhite", padding: "30px 0" }}>
+      <div>Here is some content grouped with the tab panels</div>
+      <TabPanels>
+        <TabPanel>
+          <h1>one!</h1>
+          <button>yo</button>
+        </TabPanel>
+        <TabPanel>
+          <h1>two!</h1>
+        </TabPanel>
+        <TabPanel>
+          <h1>three!</h1>
+        </TabPanel>
+      </TabPanels>
+    </div>
+  </Tabs>
+);
```

Comment:
I'm not fully convinced that this example is the best display of showing the decoupling this PR affords. I'm happy to take suggestions on better examples if people have them.

Here's a gif of the example above:

![tabs](https://user-images.githubusercontent.com/3409645/68518167-91c4a400-023f-11ea-8168-d6272ee692aa.gif)